doc/developing_oemof.rst

```diff
@@ -51,31 +51,51 @@ Newly added required packages (via PyPi) are installed by performing a manual up
 
   pip3 install --upgrade -e /path/to/the/repository
   
-Documentation
--------------
+Contribute to the documentation
```

Comment:
This section is about, where to find the developer documentation and not about how to contribute to the documentation. But if someone wants to contribute to the documentation (typos, grammar, comprehensibility) one should definitely choose the developer version. Otherwise things might be already obsolete. Maybe we could point that out that way and encourage people that every fixed typo will help.